charts/drax/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: drax
 description: Accelleran's dRAX helm chart
 type: application
-version: 7.0.0-rc.5
+version: 7.0.0-rc.6
```

Comment:
I think we should leave this out of the PR, as it avoids conflicts between different PRs that might be merged before or after each other. Once the PR is merged (and possibly others are too), we can tag a new RC.

Reply:
agreed